goal_src/jak1/kernel/gstate.gc

```diff
@@ -399,7 +399,11 @@ It type checks the arguments for the entry function.
 (defun send-event-function ((proc process-tree) (msg event-message-block))
   "Function to send an event to a process. Please use the send-event macros when possible"
   (with-pp
-    (when (and proc (!= (-> proc type) process-tree) (-> (the process proc) event-hook))
+    ;; og:preserve-this check that we have a _valid_ process. originally this just checked
+    ;; that type != process-tree, but sometimes we might get stale process ref (qbert-plat-master)
+    (when (and proc (valid? proc process #f #f 0)
```

Comment:
This is fine, but you can remove the lone `proc` here if you want, as the `valid?` call with these args will return false if the input is `#f`